src/kps/xml/objects/abstracts/BusinessEventWithLocation.java

```diff
@@ -6,9 +6,14 @@
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlTransient;
 
+/**
+ * Defines a business event which has a location.
+ */
 @XmlTransient public abstract class BusinessEventWithLocation extends BusinessEvent {
     @NotNull @XmlElement private String to = "Unknown";
     @NotNull @XmlElement private String from = "Unknown";
+
+    //Todo: Remove this shit.
```

Comment:
Yeah probably should but I cant think of a better way to do it for the time being